.github/workflows/publish-to-github-pages.yml

```diff
@@ -0,0 +1,23 @@
+name: Publish to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Generate HTML release
+        run: ./build_release_html.sh
+
+      - name: Publish generated content to GitHub Pages
+        uses: tsunematsu21/actions-publish-gh-pages@v1.0.2
+        with:
+          dir: BookHTML
+          branch: gh-pages
+          token: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
I don't think this is necessary, as we're deploying to the same repo and can use the default token.

Reply:
Indeed, I've obviously overlooked the fact that the target repository was the same as the source one.

I've updated the yaml file accordingly :)